products/stratos/modules/distribution/src/main/conf/drools/mincheck.drl

```diff
@@ -83,9 +81,8 @@ dialect "mvel"
                                  0, 0, 0, 0, 0, 0,0, 0, 0,
                                  clusterInstanceContext.getMinInstanceCount(), 0,
                                  additionalInstances, scalingReason);
-        } else {
-            log.warn("Scaling decision publisher is not enabled");
         }
+
```

Comment:
Why we are removing the else warning message?I can see this done in lot of places.